docs/modules/automation-testing/pages/scripting/create-image-injection-script.adoc

```diff
@@ -0,0 +1,98 @@
+= Create image injection script
+:navtitle: Create an image injection script
+
+Learn how to automate the process of image injection, so you can automate your test suite.

```

Comment:
What about:
Learn how to add image injection to your Appium script.

Reply:
updated